README.md

```diff
@@ -21,9 +21,14 @@ A list of hands-on opportunites (not jobs) in quantum computing
 ## Programs
 | Name | Link | Organization | Date | Status | Feedback |
 | :--: | :--: | :--: | :--: | :--: | :--: |
+| Los Alamos Quantum Computing Summer School| [Info Page](https://www.lanl.gov/projects/national-security-education-center/information-science-technology/summer-schools/quantumcomputing/index.php) | Information Science and Technology Institute (ISTI) | Summer (10 weeks) | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#los-alamos-quantum-computing-summer-school) |
```

Comment:
This already exists in the repo, so this is creating a merge conflict. See longer comment in FEEDBACK.md.